doc/ref/peer.rst

```diff
@@ -99,6 +99,36 @@ to the manage and jobs runner functions.
 .. note::
     Functions are matched using regular expressions.
 
+.. _peer-wheel:
+
+Peer Wheel Communication
+=========================
+
+Configuration to allow minions to execute wheel from the master is done via
+the ``peer_wheel`` option on the master. The ``peer_wheel`` configuration follows
+the same logic as the ``peer`` option. The only difference is that access is
+granted to wheel modules.
+
```

Comment:
I think this section could use a note about the security implications of enabling this option. Similar to the Peer Configuration section.

Reply:
Sure, but it wouldn't hurt to make that extra clear.